app/controllers/api/v3/notices_controller.rb

```diff
@@ -27,4 +28,11 @@ def create
   rescue AirbrakeApi::ParamsError
     render text: 'Invalid request', status: 400
   end
+
+  private
+
+  def authorization_token
```

Comment:
How about something like this to avoid the nil reference?

```
def authorization_token
  /Bearer (?<key>.+)/ =~ request.headers['Authorization']

  key
end
```

Reply:
I was thinking similar. This makes it a nice one-liner.

```ruby
def authorization_token
  request.headers['Authorization'][/Bearer (.+)/, 1]
end
```

Reply:
@excid3 It looks fail if `request.headers['Authorization']` is nil